test/end-to-end/offer.js

```diff
@@ -40,6 +40,7 @@ describe("Offer endpoint tests", () => {
         isHidden: false,
         isArchived: false,
         requirements: ["The candidate must be tested", "Fluent in testJS"],
+        vacancies: 2,
```

Comment:
The number of vacancies of the two offers used in tests below should be different (they both have 2 vacancies at the moment).
 https://github.com/NIAEFEUP/nijobs-be/blob/0977f3cb447acc0aa47b5e0a3b0adc040fc75bf2/test/end-to-end/offer.js#L2849-L2869 
https://github.com/NIAEFEUP/nijobs-be/blob/0977f3cb447acc0aa47b5e0a3b0adc040fc75bf2/test/end-to-end/offer.js#L2690-L2700